proj/src/store.js

```diff
@@ -0,0 +1,21 @@
+import { createStore, compose, applyMiddleware} from 'redux'
+import thunk from 'redux-thunk';
+import persistState from 'redux-localstorage'
+
+import rootReducer from 'reducers/root'
+
+const middleWare = [thunk]
+
+const initialState = {};
+
+const store = createStore(
+  rootReducer,
+  initialState,
+  compose(
+    applyMiddleware(...middleWare),
+    persistState(),
```

Comment:
does this work? you have your data saved when you refresh?

Reply:
Yep..

Reply:
👍